reference/extensions/field_expansion.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: FieldExpansion Extension
+---
+
+The `FieldExpansion` extension provides a `Controller#expanded_fields` helper for processing `:view` and/or `:fields` params to determine the final set of fields necessary to process the request.
+
+Either, or both, of the `:view` and `:fields` params may be used, provided they are defined with the proper types:
+ * `:view` must be of type `Symbol`, for example: `attribute :view, Symbol`.
+ * `:fields` must be `Praxis::Types::FieldSelector` (or a scoped subclass), for example: `attribute :fields, Praxis::Types::FieldSelector.for(Person)`.
+
+Regardless of what attributes are used, any attributes with types that are either a `MediaType` or a collection of a `MediaType` will be recursively expanded into a complete set of "leaf" attributes.
+
+
+## Expansion logic
+
+The exact expansion depends upon both the parameters defined on the action in question:
+  * If *neither* `:view` *or* `:fields` are defined: the `:default` view is expanded.
+  * If *only* `:view` is defined: the view specified in the request (or `:default` if one was not specified) is expanded.
```

Comment:
The difference is that above is covering the case where nothing is defined, this is the case where you *could* pass in `:view`. 

I'm not terribly happy with this whole section, really. It's not particularly complex logic, just involved.